### PR TITLE
Merge pull request #97 from eduardomourar/feature/external-npm-package

### PR DESCRIPTION
**Pull request by eduardomourar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,14 @@
     "url": "https://github.com/stevermeister/dutch-tax-income-calculator/issues"
   },
   "homepage": "https://github.com/stevermeister/dutch-tax-income-calculator",
+  "dependencies": {
+    "dutch-tax-income-calculator-npm": "github:eduardomourar/dutch-tax-income-calculator-npm#feature/full-calculation"
```
Comment (stevermeister) [inline package.json]: you can update this one, I've already publish the new version

File: src/calc/calc.component.js [hunk 2]
```diff
@@ -206,207 +206,13 @@ let calcComponent = {
...
-        } else {
-          if (isPercent) {
-            amount += (delta * tax);
-          } else {
-            amount = tax;
-          }
-          salary -= delta;
+        const paycheck = new SalaryPaycheck(this.salary, this.startFrom, this.year, this.ruling);
+        $log.debug(paycheck);
+        this.salary = {
```
Comment (stevermeister) [inline src/calc/calc.component.js:211]: awesome 😍

**Comment by stevermeister** [comment]
you are so quick!

File: package.json (hunk 1 above)
Comment (eduardomourar) [reply package.json]: what is the package name pushed to npm registry? or do you mean change to `github:stevermeister/dutch-tax-income-calculator-npm`?

Comment (stevermeister) [reply package.json]: yep, https://github.com/stevermeister/dutch-tax-income-calculator-npm/blob/master/package.json#L2

**Pull request merged**